projects/ngrx.io/content/guide/router-store/default-router-state-serializer.md

```diff
@@ -0,0 +1,3 @@
+# Default Router State Serializer
+
+If no router state serializer is provided through the [configuration](guide/router-store/configuration) of router store, the `DefaultRouterStateSerializer` is used. This router state serializer, serializes the URL together with the *ActivatedRouteSnapshot* from [Angular Router](https://angular.io/guide/router). The later is serialized recursively but without the recursive references.
```

Comment:
```suggestion
If no router state serializer is provided through the [configuration](guide/router-store/configuration) of router store, the `DefaultRouterStateSerializer` is used. This router state serializer, serializes the URL together with the [ActivatedRouteSnapshot](https://angular.io/api/router/ActivatedRouteSnapshot) from [Angular Router](https://angular.io/guide/router). The latter is serialized recursively but without the recursive references.
```

Reply:
What do you mean with `but without the recursive references.`?

Reply:
What I mean is that `root` and `parent` are set to `undefined` for an instance of `ActivatedRouteSnapshot` by the `serializeRoute()` method. This way a child route cannot traverse upward due to there being no recursive references. It is only possible to traverse downward in the serialized version a route. Am I wrong regarding this?

Reply:
You're correct about this. I'm wondering if we should be more explicit and say that these will be undefined instead of that it can't be traversed recursively. 

Just to be sure, did you see the suggested changes (a link to the Angular docs about the snapshot plus a typo later -> latter)?

Reply:
No, I did not but I have not. It has been a while since I have used PRs on GitHub. I will make those changes and also change the wording of the last sentence to be more explicit as you suggest.

Reply:
I have now made the changes, please proceed with the review.